server/boards/routes.ts

```diff
@@ -686,4 +688,87 @@ router.delete(
   }
 );
 
+
+/**
+ * @openapi
+ * /boards/{board_id}/roles:
+ *   get:
+ *     summary: Fetches latest roles summary for the board.
+ *     operationId: getBoardRolesByExternalId
+ *     tags:
+ *       - /boards/
+ *     security:
+ *       - {}
+ *     parameters:
+ *       - name: board_id
+ *         in: path
+ *         description: The id of the board.
+ *         required: true
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *     responses:
+ *       200:
+ *         description: The board roles summary.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: "#/components/schemas/RealmRoles"
+ *             examples:
+ *               twisted_minds:
+ *                 value:
+ *                   roles:
+ *                     - user_id: "1"
```

Comment:
role id is a UUIDs, and should be kept as such in examples because (i'm pretty sure) OpenAPI can test for them being such, and it would be an error to return the internal ones.

If you're using VSCode, there's an extension to generate them: https://marketplace.visualstudio.com/items?itemName=netcorext.uuid-generator